library/ssl_tls13_server.c

```diff
@@ -1701,6 +1749,17 @@ static int ssl_tls13_postprocess_client_hello( mbedtls_ssl_context* ssl )
         return( ret );
     }
 
+#if defined(MBEDTLS_SSL_EARLY_DATA)
+    /* There is enough information, update early data state. */
+    ssl_tls13_update_early_data_state( ssl );
+#elif defined(MBEDTLS_DEBUG_C)
+    if( !mbedtls_ssl_tls13_early_data_is_rejected( ssl ) &&
+        mbedtls_ssl_tls13_early_data_is_received( ssl ) )
```

Comment:
Not sure what case you mean to check here, but currently if `early_data_state` is set to `xxx_ACCEPTED`, `"should never happend."` would be printed.

Reply:
It has been removed in my local version. will update it.